src/component/AuthForms/LoginForm.jsx

```diff
@@ -1,21 +1,55 @@
 import React from "react";
 import "./AuthForms.css";
-import { Link } from "react-router-dom";
+import { Link, useNavigate } from "react-router-dom";
+import { useState } from "react";
+import axios from "axios";
+import { useAuthContext } from "../../Context";
 
 const LoginForm = () => {
+
+    const [formData, setFormData] = useState({email: "", password: ""})
+    const navigate = useNavigate()
+    const{token, setToken} = useAuthContext()
+
+
+    const loginFormHandler = (e) => {
+        e.preventDefault()
+         
+        const loginUser = formData;
+
+        try {
+            (async () => {
```

Comment:
good use of ify

Reply:
You mean [IIFE](https://developer.mozilla.org/en-US/docs/Glossary/IIFE). ok got it Thank you